pwiz_tools/Shared/CommonUtil/SystemUtil/PathEx.cs

```diff
@@ -172,14 +172,50 @@ public static String GetDirectoryName(String path)
         /// <param name="path">Path to convert to long-path syntax. This must be a fully qualified path.</param>
         public static string ToLongPath(this string path)
         {
-            // Note that Path.GetFullPath() will be equivalent to
-            // Path.Combine(Directory.GetCurrentDirectory(), path) for a relative path,
-            // which is rarely appropriate. So, using this function requires an already
+            if (path.StartsWith(PREFIX_LONG_PATH))
+                return path;
+
+            // Note: Using this function requires an already
             // fully qualified path.
-            if (!Equals(path, Path.GetFullPath(path)))
+            if (!IsPathFullyQualified(path))
                 throw new ArgumentException($@"Failed attempting to use long-path syntax for the path '{path}' which is not fully qualified.");
+
             // Avoid adding the long-path prefix to a path that already has it.
-            return path.StartsWith(PREFIX_LONG_PATH) ? path : PREFIX_LONG_PATH + path;
+            return PREFIX_LONG_PATH + path;
+        }
+
+        /// <summary>
+        /// Returns a true if the path is fully qualified
+        /// </summary>
+        /// <param name="path">Path to convert to long-path syntax. This must be a fully qualified path.</param>

```

Comment:
This part of the comment isn't correct.
The `<param>` tags are not required in these comments so you can just remove it.